### ISSUE TITLE
 i/j switched in cmor ocean files

### ISSUE THREAD

**Issue by pabretonniere** [goord/ece2cmor3#73]
Hi @goord ,

We discovered this morning a strange issue in the cmor files from the ocean of our simulations. I checked the files that you uploaded to Jasmin and they have the same "issue". 
Below is the header of a tos file that I processed. As you can see, i < j and then corresponds to the y/latitude axis which is a bit counter intuitive. All the rest is coherent, meaning that it is tos(time,i,j)
and that if you plot the file, it will look good, the lat and lon won't be switched so the file itself can't be considered as erroneous. The only issue is that, apart from being counter intuitive, the Nemo original files are stored with x being x (horizontal axis) and y being y and in addition, that some softwares (we discovered the issue using the cdf tools) are reading i and j assuming i is the horizontal index and j the vertical one, making the tool fails if they are switched. 
I've investigated a bit and all the Nemo files processed with ece2cmor/cmor3 have the issue whereas Nemo files processed with cmor2 don't have it. Also, I couldn't find anywhere in ece2cmor or the MIP tables where the assignation of i and j to x or y was done, so I guess this comes from cmor3 itself. Could you please confirm me that in all your files you have the same behaviour? If yes, I will open an issue on the cmor github but I'd like to make sure this is not an ece2cmor problem before.

Thanks a lot,
Best regards,

Pierre-Antoine

```

netcdf tos_Omon_EC-Earth3-HR_historical_r1i1p1f1_gn_195001-195001 {
dimensions:
        time = UNLIMITED ; // (1 currently)
        i = 1050 ;
        j = 1442 ;
        bnds = 2 ;
        vertices = 4 ;
variables:
        double time(time) ;
                time:bounds = "time_bnds" ;
                time:units = "days since 1900-01-01 00:00:00" ;
                time:calendar = "gregorian" ;
                time:axis = "T" ;
                time:long_name = "time" ;
                time:standard_name = "time" ;
        double time_bnds(time, bnds) ;
        int i(i) ;
                i:units = "1" ;
                i:long_name = "cell index along first dimension" ;
        int j(j) ;
                j:units = "1" ;
                j:long_name = "cell index along second dimension" ;
        float latitude(i, j) ;
                latitude:units = "degrees_north" ;
                latitude:missing_value = 1.e+20f ;
                latitude:_FillValue = 1.e+20f ;
                latitude:bounds = "vertices_latitude" ;
        float longitude(i, j) ;
                longitude:units = "degrees_east" ;
                longitude:missing_value = 1.e+20f ;
                longitude:_FillValue = 1.e+20f ;
                longitude:bounds = "vertices_longitude" ;
        float vertices_latitude(i, j, vertices) ;
                vertices_latitude:units = "degrees_north" ;
                vertices_latitude:missing_value = 1.e+20f ;
                vertices_latitude:_FillValue = 1.e+20f ;
        float vertices_longitude(i, j, vertices) ;
                vertices_longitude:units = "degrees_east" ;
                vertices_longitude:missing_value = 1.e+20f ;
                vertices_longitude:_FillValue = 1.e+20f ;
        float tos(time, i, j) ;
                tos:standard_name = "sea_surface_temperature" ;
                tos:long_name = "Sea Surface Temperature" ;
                tos:comment = "temperature of liquid ocean.  Note that
the correct standard_name for this variable is
\'sea_surface_temperature\', not \'surface_temperature\', but this was
discovered too late to correct.  To maintain consistency across CMIP5
models, the wrong standard_name will continue to be used." ;
                tos:units = "K" ;
                tos:original_name = "tos" ;
                tos:original_units = "degC" ;
                tos:history = "2017-11-14T09:20:08Z altered by CMOR:
Converted units from \'degC\' to \'K\'. 2017-11-14T09:20:08Z altered by
CMOR: Reordered dimensions, original order: i j time." ;
                tos:cell_methods = "time: mean" ;
                tos:cell_measures = "area: areacello" ;
                tos:missing_value = 1.e+20f ;
                tos:_FillValue = 1.e+20f ;
                tos:coordinates = "latitude longitude" ;

```

**Comment by goord** [comment]
Hi Pierre-Antoine
I have the same issue but it's not a cmor issue, it is our nemo grid definition in combination with the reordered dimensions (see history) that causes this.

**Comment by goord** [comment]
Fixed in master and primavera branches f5eded19b2fe2fa26f0c5e96ddc355bc0c45bae8
